### ISSUE TITLE
installation error, spacepy is required?

### ISSUE THREAD

**Issue by JouleCai** [MAVENSDC/cdflib#122]
env: annaconda python=3.8 ubuntu

```
pip install cdflib
Requirement already satisfied: cdflib in ./.local/lib/python3.8/site-packages (0.3.20)
Collecting numpy
  Using cached numpy-1.21.1-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (15.8 MB)
Collecting attrs>=19.2.0
  Using cached attrs-21.2.0-py2.py3-none-any.whl (53 kB)
Installing collected packages: numpy, attrs
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
spacepy 0.2.2 requires h5py>=2.6, which is not installed.
spacepy 0.2.2 requires matplotlib>=1.5, which is not installed.
spacepy 0.2.2 requires networkx>=1.0, which is not installed.
spacepy 0.2.2 requires python-dateutil>=1.4, which is not installed.
spacepy 0.2.2 requires scipy>=0.11, which is not installed.
Successfully installed attrs-21.2.0 numpy-1.21.1
```

**Comment by dstansby** [comment]
This looks like (potentially) an error with `spacepy` - it looks like `cdflib` installed fine, but from some previous installations you have `spacepy` installed but not some of the `spacepy` dependencies. I would suggest opening an issue at https://github.com/spacepy/spacepy/issues to see if they can help there.

**Comment by JouleCai** [comment]
Thank you. 

I realised that I installed spacepy in the conda base long time ago. When I installed cdflib in a new virtual env, the messages were prompted out. 

It seems that spacepy in the conda base affect the installation in the virtual environment. However, for my understanding, the conda environments should not inherit base packages. 

The issue was solved after I uninstalled spacepy in the conda base.
